benchmarks/nemo/nemolite2d/manual_versions/psykal_cpp/time_step_omp.cpp

```diff
@@ -115,7 +115,8 @@ extern "C" void c_invoke_time_step(
     }
 
     // Boundary conditions bc_flather_v kernel (whole domain but top y boundary)
-    #pragma omp parallel for
+    // We cannot execute this loop in (OpenMP) parallel because of the
```

Comment:
I think it is mainly to have a fair comparison with the Fortran OpenMP, but it should have a small impact as flather takes a small percentage of execution time.

Aha, the psyclone version does omp-parallelise this loop, so this will need a 'kernel fix' as you mention at some point.